src/components/withWindowDimensions/getWindowHeightAdjustment.android.js

```diff
@@ -0,0 +1,5 @@
+// On Android the window height does not include the status bar height, so we need to add it manually.
+export default function getWindowHeightAdjustment(insets) {
```

Comment:
Should we rename this method? We are not passing any height related info and there's no adjustment made. Could this be `getWindowInsets` or something ?

Reply:
I wanted to make the function name more specific to this component use case to avoid confusion where someone would be tempted to use it for something else. Since on all platforms except android it returns 0 and not window insets it might be confusing?